src/app/main.cpp

```diff
@@ -39,6 +41,34 @@ void HandleStop(int i) {
   }
 }
 
+// Print a path warning if the running directory is NOT one level above the binary directory
+void printPathWarning() {
+  char cwd[PATH_MAX];
+  getcwd(cwd, PATH_MAX);
+  std::string currentWorkingDir = cwd;
+
+  char binaryPathRaw[PATH_MAX];
+  int count = readlink("/proc/self/exe", binaryPathRaw, PATH_MAX);
+
+  if (count == -1) {
+    // We failed getting the binary path, quit
+    return;
+  }
+  std::string binaryPath = std::string(binaryPathRaw, count);
+
+  // Get the offset to check
+  unsigned int offset = binaryPath.size() - (binaryPath.size() - currentWorkingDir.size());
```

Comment:
Change `unsigned int offset = binaryPath.size() - (binaryPath.size() - currentWorkingDir.size());` to `unsigned int offset = currentWorkingDir.size();`.
It does the same, just simplified.